docs/cloud/aws/budget-actions/budget-actions.md

```diff
@@ -55,16 +55,16 @@ This managed policy is focused on specific actions that AWS Budgets takes on you
 11. For **Do you want to automatically run this action when this threshold is exceeded**, choose **Yes** or **No**. If you choose No, then you run the action manually on the **Alert details page**. For instructions, see Reviewing and approving your budget action.
 
 12. For **How do you want to be alerted when this action is run, choose Use the same alert settings when you defined this threshold** or **Use different alert settings**. To use different alert settings, complete the **Notification preferences** specific to this action.
-    ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/7.png)
+    ![](images/7.png)
 
 
 13.	Choose **Next**.
 
 14.	Review your budget settings, and then choose **Create budget**.
-    ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/8.png)
+    ![](images/8.png)
```

Comment:
Something weird is happening here in the rendering. For all the images you work with, I would make sure the rendering looks good. 

![Screenshot 2024-09-06 at 4 19 52 PM](https://github.com/user-attachments/assets/3f867cd6-defc-4358-a940-b09b9a057af6)

Reply:
<img width="908" alt="Screenshot 2024-09-06 at 4 23 30 PM" src="https://github.com/user-attachments/assets/c7e37b4c-1c99-40c0-9303-e8ccb0d65de8">

Reply:
b9fea6a

Reply:
Fixed by adding some new lines

Reply:
11d9cc0

Reply:
Final fix ^^